formula/src/test/java/com/instacart/formula/ActionTest.kt

```diff
@@ -195,6 +207,22 @@ class ActionTest {
         }
     }
 
+    @Test
+    fun `fromFlow - error is emitted`() = runTest {
+        val exception = RuntimeException("Test exception")
+        val action = Action.fromFlow<Unit> {
+            flow {
+                throw exception
+            }
+        }
+        val result = runCatching {
+            action.test {}
+        }
+        assertThat(result.exceptionOrNull()).hasMessageThat().isEqualTo(
+            "Expected no errors, but got: [java.lang.RuntimeException: Test exception]"
+        )
```

Comment:
nit: add test for skipping `CancellationException`?

Reply:
Good call, added